docs/advanced/command-line/files/feedback.md

```diff
@@ -0,0 +1,15 @@
+# Feedback
+This file describes the feedback instances, one feedback instance per line. 
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **feedback.csv** in the Manual Books example.
+
+### Headers
+The headers in this file should match the names and the order of the fields of the target schema, as defined in the property product.fields(see [Wrangle Properties](../commands.md)). 
```

Comment:
a space is needed "product fields (see..."

Reply:
Done!